src/statusTile/updateStatusTileScope.test.js

```diff
@@ -0,0 +1,31 @@
+jest.mock('../atomInterface');
+jest.mock('../editorInterface');
+
+const { getCurrentScope } = require('../editorInterface');
+const { getAllScopes } = require('../atomInterface');
+const updateStatusTileScope = require('./updateStatusTileScope');
+
+const callUpdateStatusTileScope = () => {
+  const div = { dataset: {} };
+  const editor = {};
+
+  updateStatusTileScope(div, editor);
+
+  return { div, editor };
+};
+
+beforeEach(() => {
+  getAllScopes.mockImplementation(() => ['source.js']);
+});
+
+it('sets the match-scope data attribute to true if the editor is in scope', () => {
+  getCurrentScope.mockImplementation(() => 'source.js');
+  const { div } = callUpdateStatusTileScope();
+  expect(div.dataset.prettierMatchScope).toBe('true');
```

Comment:
can we separate the "given, when, then" parts of the test with blank lines, just for readability?